src/meta/expr.lean

```diff
@@ -120,6 +120,26 @@ meta def is_num_eq : expr → expr → bool
 | `(%%a/%%a') `(%%b/%%b') :=  a.is_num_eq b
 | _ _ := ff
 
+meta def is_eq_or_iff_after_binders : expr → bool
```

Comment:
Why is this function needed?

Reply:
Actually, I think we could remove it for now. It is eventually needed for an extension of `rewrite_search` that inspects the local environment to discover relevant rewrite lemmas, but this won't be in the initial PR. I'd suggest we don't put this one into mathlib for now, to keep things simple.

Reply:
I've removed it.